### PR TITLE
Fdomf/docs

### PR DESCRIPTION
**Pull request by fdomf**
#### Description

Updated documentation

#### Where has this been tested?

  * **Platform(s):** Ubuntu 16.04
  * **Python version(s):** ...
  * **Unreal Engine version(s):** ...

---
This change is [Reviewable](https://reviewable.io/reviews/carla-simulator/carla/2030)

### CODE REVIEW

File: Docs/python_api.md [hunk 1]
```diff
@@ -1925,13 +1925,13 @@ Lidar sensor measurement data.
 - <a name="carla.LidarMeasurement.horizontal_angle"></a>**<font color="#f8805a">horizontal_angle</font>** (_float_)  
 Horizontal angle that the Lidar has rotated at the time of the measurement.  
 - <a name="carla.LidarMeasurement.channels"></a>**<font color="#f8805a">channels</font>** (_int_)  
-Number of lasers.  
+The lasers' number.
```
Comment (ThrashAbaddon) [inline Docs/python_api.md]: no

File: Docs/python_api_tutorial.md [hunk 2]
```diff
@@ -426,11 +426,35 @@ waypoint = map.get_waypoint(vehicle.get_location())
...
 
+Waypoints have their unique identifier [`carla.Waypoint.id`](python_api.md#carla.Waypoint.id) based on the hash of its
+[`road_id`](python_api.md#carla.Waypoint.road_id), [`section_id`](python_api.md#carla.Waypoint.section_id),
+[`lane_id`](python_api.md#carla.Waypoint.lane_id) and [`s`](python_api.md#carla.Waypoint.s).
+They also provide more information about lanes, such as the [`lane_type`](python_api.md#carla.Waypoint.lane_type) of the current waypoint
+and if a [`lane_change`](python_api.md#carla.Waypoint.lane_change) is possible and in which direction.
+
+```py
+# Nearest waypoint on the center of a Driving or Sidewalk lane.
+waypoint = map.get_waypoint(vehicle.get_location(),project_to_road=True, lane_type=(carla.LaneType.Driving | carla.LaneType.Sidewalk))
```
[the remark this answers is not in the document]
Comment (fdomf) [reply Docs/python_api_tutorial.md]: With project_to_road flag – If True, the waypoint will be at the center of the nearest lane given a location

File: Docs/python_api.md [hunk 3]
```diff
@@ -1214,24 +1214,24 @@ Map name. Comes from the Unreal's UMap name if loaded from a Carla server.
...
         - `lane_type` (_[carla.LaneType](#carla.LaneType)_) – This parameter is used to limit the search on a certain lane type. This can be used like a flag: `LaneType.Driving & LaneType.Shoulder`.  
     - **Return:** _[carla.Waypoint](#carla.Waypoint)_  
 - <a name="carla.Map.get_topology"></a>**<font color="#7fb800">get_topology</font>**(<font color="#00a6ed">**self**</font>)  
-It provides a minimal graph of the topology of the current OpenDRIVE file. It is constituted by a list of pairs of waypoints, where the first waypoint is the origin and the second one is the destination. It can be loaded into [NetworkX](https://networkx.github.io/). A valid output could be: `[ (w0, w1), (w0, w2), (w1, w3), (w2, w3), (w0, w4) ]`.  
+It provides a minimal graph of the topology of the current OpenDRIVE file. It is constituted by a waypoint's pair list, where the first waypoint is the origin and the second one is the destination. It can be loaded into [NetworkX](https://networkx.github.io/). A valid output could be: `[ (w0, w1), (w0, w2), (w1, w3), (w2, w3), (w0, w4) ]`.
```
[the remark this answers is not in the document]
Comment (fdomf) [reply Docs/python_api.md]: I'll check all descriptions

File: Docs/python_api.md [hunk 4]
```diff
@@ -1214,24 +1214,24 @@ Map name. Comes from the Unreal's UMap name if loaded from a Carla server.
 - <a name="carla.Map.__init__"></a>**<font color="#7fb800">\__init__</font>**(<font color="#00a6ed">**self**</font>, <font color="#00a6ed">**name**</font>, <font color="#00a6ed">**xodr_content**</font>)  
 Constructor for this class useful if you want to use a `XODR` (OpenDRIVE) file without importing it from any Carla server running.  
     - **Parameters:**
-        - `name` (_str_) – Name of the current map.  
+        - `name` (_str_) – The map's current name.  
         - `xodr_content` (_str_) – XODR content as string.  
     - **Return:** _list([carla.Transform](#carla.Transform))_  
 - <a name="carla.Map.get_spawn_points"></a>**<font color="#7fb800">get_spawn_points</font>**(<font color="#00a6ed">**self**</font>)  
-Returns a list of transformations corresponding to the recommended spawn points over the map.  
+Returns a [carla.Transform](#carla.Transform)'s list corresponding to the recommended spawn points over the map.
```
[the remark this answers is not in the document]
Comment (fdomf) [reply Docs/python_api.md]: I'll redo all changes. Thanks for the review!

**Pull request closed without merging**